rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -237,19 +255,21 @@ class DebruijnInterpreter[M[_], F[_]](
 
     val starts = jobs.map(_.size).scanLeft(0)(_ + _).toVector
 
-    jobs.zipWithIndex.parTraverse_ {
-      case (job, jobIdx) => {
-        def mkRand(termIdx: Int): Blake2b512Random =
-          split(starts.last, starts(jobIdx), rand)(termIdx)
-        job.run(mkRand)(env, sequenceNumber)
+    spanM.trace(parSpanLabel) {
```

Comment:
I agree that the code above most likely executes fast, but I'd rather make the span wider than be surprised later. Options: 

1. Ideally, the span would cover the whole method body. This could be achieved by moving the nested methods out of the `eval` method and making them private.
 - Similar move could be made for `val(mat: Match)`, but feel free to skip it 

2. I'm fine if you just widen this span to cover `val filteredExprs` and `val jobs = List(` though.

Reply:
It's enough to move the code from current line between lines 230 and 231